src/components/form/templates/array-field-collapsible-item.ts

```diff
@@ -38,6 +38,8 @@ export class CollapsibleItemTemplate extends React.Component {
     constructor(public props: CollapsibleItemProps) {
```

Comment:
For the changelog, it would be helpful to have a scope in the commit message:

`fix(form): prevent argument mismatching in case of value being an array`